workers/unity/Packages/com.improbable.gdk.transformsynchronization/TransformUtils.cs

```diff
@@ -2,60 +2,324 @@
 
 namespace Improbable.Gdk.TransformSynchronization
 {
+    /// <summary>
+    ///     A collection of utility functions for use with the Transform Synchronization Feature Module.
+    /// </summary>
     public static class TransformUtils
```

Comment:
Should everything in here be `public`? Or can some be internal?

Reply:
the only things left public now are:
- `CreateTransformSnapshot` using `Vector3` for location
- `CreateTransformSnapshot` using `Coordinates` for location
- `ToCoordinates` from `Vector3`